src/index.html

```diff
@@ -11,6 +15,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="get"
+      class="form"
+      data-qa="big"
+    >
+      <label class="searchInput">
+        <img
+          src="/images/Search.svg"
+          class="icon"
+          alt="bigSearchIcon"
+        >
```

Comment:
Better to use background image for decorative icon

Reply:
done